src/lib/proxie.ts

```diff
@@ -0,0 +1,339 @@
+const inquirer = require('inquirer');
+const fuzzy = require('fuzzy');
+import getPort from 'get-port';
+import { Answers } from 'inquirer';
+import CloudSqlProxyCli from './CloudSqlProxyCli';
+import GcloudCli, { Cluster, Project, SqlInstance, VMInstance } from './GcloudCli';
+import KubectlCli, { Pod } from './KubectlCli';
+const commandExists = require('command-exists');
+
+export const globalOptions = {
+    debug: !!/bescripts/.test(String(process.env.DEBUG)),
+};
+
+const gcloud = new GcloudCli(globalOptions);
+const kubectl = new KubectlCli(globalOptions);
+const cloudSqlProxy = new CloudSqlProxyCli(globalOptions);
+
+inquirer.registerPrompt('autocomplete', require('inquirer-autocomplete-prompt'));
+
+const initializationCheck = async () => {
+    console.log('Command check:');
+    const missing = (await Promise.all(['gcloud', 'kubectl', 'cloud_sql_proxy']
+        .map(async cmd => {
+            let status = '❌';
+            try {
+                await commandExists(cmd);
+                status = '√';
+            } catch (error) {}
+            console.log(` ${cmd} ${status}`);
+            return { status, cmd };
+        })
+    ))
+        .filter(({ status }) => status !== '√');
```

Comment:
Extracting this symbol would be slightly safer for future

Reply:
Refactored, used the log symbols.